lib/api_2captcha/client.rb

```diff
@@ -5,7 +5,7 @@
 require 'base64'
 require 'open-uri'
 
-class Api2Captcha::Client
+class Client
```

Comment:
@PBadicean pls check the PR: https://github.com/2captcha/2captcha-ruby/pull/6 that has a suggestion for this. Probably it's better.